src/chains/ethereum/options/src/fork-options.ts

```diff
@@ -160,6 +172,26 @@ export type ForkConfig = {
       type: number;
       hasDefault: true;
     };
+
+    /**
+     * Don't cache forking requests in a persistent db.
```

Comment:
Disables caching of all forking requests.
```suggestion
     * Disables caching of all forking requests.
```